### PR TITLE
Uhf 8529 childcare calculator text updates

### PR DESCRIPTION
**Pull request by Arkkimaagi**
# [UHF-8529](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8529)

* We have some text and translation updates for "Early childhood education fee calculator"
    * Ingress updates
    * Translation file updates
    * Sibling discount as new detail to the receipt

## What was done

* [Demo environment](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/HEL/pages/8189902851/Demo-ymp+rist+n+k+ytt) was updated with new ingress in three languages
* Translation file was verified and updated
* Sibling discount feature was added

## How to install

* Make sure your KASKO instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-8529_childcare_calculator_text_updates`
* Run `make drush-cr`
* Add a page with the early childhood calculator.

## How to test

* [x] Add a family with size of at least 5
* [x] Fill details for at least 4 children
* [x] Notice that child 2 has 40% sibling discount row added, children 3 onwards have sibling discount of 20% row added
* [ ] Check that texts match what the JIRA issue [UHF-8529](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8529) describes
* [x] Check that [demo site](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/HEL/pages/8189902851/Demo-ymp+rist+n+k+ytt) ingress texts match what the JIRA issue [UHF-8529](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8529) describes
* [ ] Check that code follows our standards

## Designers review

* [X] This PR does not need designers review

[UHF-8529]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8529?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by teroelonen** [comment]
The full-time texts are a bit strange. There shouldn't be so much space there between the dash I think.
Screenshot 2023-05-25 at 17 16 25

**Comment by teroelonen** [comment]
Also for the second child there is slightly different spacing. 
Screenshot 2023-05-25 at 17 18 09

File: src/js/calculator/early_childhood_education_fee/_translations.js [hunk 1]
```diff
@@ -57,14 +57,24 @@ const translations = {
...
-    sv: 'Nästa barn i åldersordningen',
-    en: 'Second youngest child in early childhood education',
+    fi: 'Seuraavaksi nuorin lapsi',
+    sv: 'Näst yngsta barnet',
+    en: 'Next youngest child',
+  },
+  second_youngest_child_sibling_discount: {
+    fi: 'Sisarusalennus, ${discount} % nuorimman lapsen kokoaikaisesta maksusta.',
+    sv: 'Syskonrabatt, ${discount} % av heltidsavgiften för det yngsta barnet.',
+    en: 'Sibling discount, ${discount} % of the full- time fee for the youngest child.',
```
Comment (teroelonen) [inline src/js/calculator/early_childhood_education_fee/_translations.js:72]: Fix the full-time here.

**Review by teroelonen — CHANGES_REQUESTED** [review]
Check the comments - otherwise works very nicely!

File: src/js/calculator/early_childhood_education_fee/_translations.js (hunk 1 above)
Comment (Arkkimaagi) [reply src/js/calculator/early_childhood_education_fee/_translations.js]: Good catch! Fixed now!

File: src/js/calculator/early_childhood_education_fee/_translations.js [hunk 2]
```diff
@@ -57,14 +57,24 @@ const translations = {
...
+  },
+  nth_youngest_child_sibling_discount: {
+    fi: 'Sisarusalennus, ${discount} % nuorimman lapsen kokoaikaisesta maksusta.',
+    sv: 'Syskonrabatt, ${discount} % av heltidsavgiften för det yngsta barnet.',
+    en: 'Sibling discount, ${discount} % of the full - time fee for the youngest child.',
```
[the remark this answers is not in the document]
Comment (Arkkimaagi) [reply src/js/calculator/early_childhood_education_fee/_translations.js]: Good catch! Fixed now!

**Review by teroelonen — APPROVED** [review]
This works nicely, great job!! 🦖

**Pull request merged**